util/setup_dev.sh

```diff
@@ -179,6 +179,16 @@ if [[ -n "$NGX_WASM_RUNTIME" ]] && ! [[ -n "$NGX_WASM_RUNTIME_LIB" ]]; then
     $NGX_WASM_DIR/util/runtime.sh -R "$NGX_WASM_RUNTIME"
 fi
 
+if [[ ! -d "$DIR_PROXY_WASM_GO_SDK" ]]; then
+    notice "downloading proxy-wasm go SDK..."
+    $NGX_WASM_DIR/util/sdk.sh -S go --clean
+fi
+
+if [[ ! -d "$DIR_PROXY_WASM_ASSEMBLYSCRIPT_SDK" ]]; then
+    notice "downloading proxy-wasm AssemblyScript SDK..."
+    $NGX_WASM_DIR/util/sdk.sh -S assemblyscript --clean
+fi
```

Comment:
Do we have somewhere the checks we discussed at length? Namely, that if a machine does not have Node.js or TinyGo installed, these setups should be skipped? The part I was the most excited about with this PR was that we would stop requiring TinyGo (and don't add Node.js) from the list of "hard requirements"; the goal being to ease-up development onboarding for the module. Did I miss something that does this in the changeset?